common/components/deploy_model/asset.yaml

```diff
@@ -1,2 +1,3 @@
 type: component
-spec: spec.yaml
\ No newline at end of file
+spec: spec.yaml
+categories: ["Models"]
```

Comment:
@skasturi categories section for the modified assets is now required. Please confirm "Models" is the right category to use for these changes.

Reply:
@esrao I see categories for environment only. Do we need to add for component and model spec files as well?
cc: @elesel

Reply:
Ohh I see new changes requires all assets to have categories

Reply:
I think "Models" makes sense for this component because it's a component that works with models.